.github/workflows/.deploy.yaml

```diff
@@ -45,7 +45,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - uses: andstor/file-existence-action@v2
+        id: check_files
+        with:
+          files: "apps/${{ github.workflow }}/.nais/prod.yaml"
```

Comment:
Alternativt uten plugin: 

```
      - id: check_files
         run: if [ -f "apps/${{ github.workflow }}/.nais/prod.yaml" ]; then echo "true" >> $GITHUB_OUTPUT; fi
```

eller 

```
      - id: check_files
        run: |
          [ -f "apps/${{ github.workflow }}/.nais/prod.yaml" ] && echo "true" >> $GITHUB_OUTPUT
```

... ikke testet 😅

Reply:
Jeg tar en rask sjekk av den actionen for å se om den virker fornuftig, hvis ikke kan vi skrive om som det forslaget 👍